src/app/leverage/components/portfolio-widget/portfolio-widget.tsx

```diff
@@ -29,9 +28,21 @@ const OpenPositions = () => {
   const { chainId } = useNetwork()
   const [selectedIndex, setSelectedIndex] = useState(0)
   const { queryParams, updateQueryParams } = useQueryParams()
+  const fetchPositions = useSetAtom(fetchPositionsAtom)
```

Comment:
Would it make sense to create a `useSetPositions` or something like this here? Just not to sprinkle atom imports all over the project in case we ever change the library.

Reply:
We could do that, but I'm not sure if at an event of a library change we will be at an advantage if we do because then you need to wrap all of these in its own function, and instead of the imports you have to edit those files, which doesn't sound much less of a work for me. Still if you feel like we should do this let me know